#67 - adding the mission statement (#85)
tango-controls/rfc: README.md

This repository is the home of all Tango Open Specification.

## Mission

> The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:

Author: Add version number

> The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:

Author: ```suggestion
The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:
```

> The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:

Author: @andygotz and/or @bourtemb, I have updated this with the version number according to what we talk on Monday.
Let me know if it is ok, so I can merge this PR.

Reviewer: I would just specify the version as "V9 LTS"

Author: @andygotz, thanks, done. Please, let me know if it is now ok.

> The goal of this RFC project is to provide a formal specification of the current (v9.3 lts) Tango Controls system. This specification shall include:

Author: ```suggestion
The goal of this RFC project is to provide a formal specification of the current (V9 LTS) Tango Controls system. This specification shall include:
```

> 1. concepts

Author: ```suggestion
1. concepts,
```

2. terminology, 
3. protocol behaviour,
4. conventions,

> each on a sufficient level for future evolution of Tango Controls and/or implementation in other languages. In that respect, concepts are more important than implmentation details.

Reviewer 2: ```suggestion
each on a sufficient level for future evolution of Tango Controls and/or implementation in other languages. In that respect, concepts are more important than implementation details.
```

This repository is the home of all Tango Open Specification.

## Mission

The goal of this RFC project is to provide a formal specification of the current Tango Controls system. This specification shall include:

1. concepts
2. terminology, 
3. protocol behaviour,
4. conventions,

> 5. common APIs,

Author: ```suggestion
```

> 6. common utilities,

Author: ```suggestion
5. common utilities,
```

> each on a sufficient level for preparation of specification of a new vesrion of Tango Controls (Tango v10). In that respect concepts are more important than details.

Author: ```suggestion
each on a sufficient level for future evolution of Tango Controls. In that respect, concepts are more important than details.
```
